### DISCUSSION TITLE
Adjustable poinz

### DISCUSSION THREAD

**Discussion by claudiogar** [Zuehlke/poinz#296] [Q&A] [answered]
We use Poinz not only for estimating stories but, as part of the SAFe process, also to provide confidence votes that do not follow Fibonacci (i.e. values from 1 to 5).
It would be nice to have the option to choose between a Fibonacci mode (default) and a "normal" mode.

EDIT: just realized that there is the option to add Custom options. That should be enough.

**Comment by xeronimus** [comment, marked as answer]
Hi @claudiogar ,

Exactly,  I also would have suggested the "Custom Cards" feature:  https://github.com/Zuehlke/poinz/blob/master/docu/manual.md#custom-cards
:)
